sdk/communication/Azure.Communication.Chat/src/ChatThreadClient.cs

```diff
@@ -375,67 +416,101 @@ public virtual Response AddMembers(IEnumerable<ChatThreadMember> members, Cancel
             }
         }
 
-        /// <summary> Gets the members of a thread asynchronously. </summary>
+        /// <summary> Gets the participants of a thread asynchronously. </summary>
+        /// <param name="skip"> Skips participants up to a specified position in response. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
-        public virtual AsyncPageable<ChatThreadMember> GetMembersAsync(CancellationToken cancellationToken = default)
+        public virtual AsyncPageable<ChatParticipant> GetParticipantsAsync(int? skip = null, CancellationToken cancellationToken = default)
```

Comment:
Do we need to expose the `skip` field? Sounds more like capability that Azure.Core might want to add to pageable objects in general. So I recommend not adding it to the public interface unless we have a good reason.